analyzers/src/SonarAnalyzer.CSharp/Rules/DoNotShiftByZeroOrIntSize.cs

```diff
@@ -139,8 +138,8 @@ protected override void Initialize(SonarAnalysisContext context)
         private static bool TryGetConstantValue(ExpressionSyntax expression, out int value)
         {
             value = 0;
-            return expression?.RemoveParentheses() is LiteralExpressionSyntax literalExpression &&
-                int.TryParse(literalExpression?.Token.ValueText, out value);
+            return expression?.RemoveParentheses() is LiteralExpressionSyntax literalExpression
```

Comment:
It is odd that the coverage did not improve even if we improved it. On the good side I took a closer look at this return statement and realized that the null conditional access is not needed neither on expression neither on literalExpression.